src/components/marquee-wizard/marquee-wizard.component.spec.ts

```diff
@@ -0,0 +1,268 @@
+import { Component, OnDestroy, ChangeDetectorRef } from '@angular/core';
+import {
+    async,
+    ComponentFixture,
+    fakeAsync,
+    TestBed,
+    tick
+} from '@angular/core/testing';
+import { MarqueeWizardModule } from './marquee-wizard.module';
+
+@Component({
+    selector: 'marquee-wizard-app',
+    template: `
+        <ux-marquee-wizard
+            class="marquee-wizard"
+            [(step)]="step"
+            [description]="description"
+            (onFinish)="close()"
+            (onCancel)="close()"
+            [previousVisible]="step !== 0"
+            [cancelVisible]="false"
+        >
+            <ng-template #description>
+                <img
+                    alt="Icon"
+                    src="https://pages.github.houston.softwaregrp.net/caf/ux-aspects-micro-focus/docs/app/assets/img/marquee-wizard-icon.svg"
+                />
+
+                <h3 id="marquee-wizard-title" class="marquee-title">
+                    Marquee Wizard
+                </h3>
+                <p class="m-b-nil">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                    Aenean sodales lacus vitae congue lacinia. Phasellus finibus
+                    dolor efficitur quam vestibulum feugiat.
+                </p>
+            </ng-template>
+
+            <ux-marquee-wizard-step
+                *ngFor="let step of steps; index as pageIndex"
```

Comment:
I don't think index is used here, so it can be removed. Also, index should be used like `let pageIndex = index`.